application/invites/invites_view.py

```diff
@@ -0,0 +1,32 @@
+"""Handle requests on the team resource."""
+from flask import abort
+from flask_apispec import use_kwargs, marshal_with
+from flask_apispec.views import MethodResource
+from flask_jwt_extended import get_jwt_identity, jwt_required
+
+from application.teams.team import Team
+from application.teams.team_schema import TEAM_SCHEMA
+from application.users.user import User
+from .invites_validator import accept_invite_args
+
+
+class AcceptInvite(MethodResource):

```

Comment:
Why do you not call it InvitesView?

Reply:
But I get that then the post method would not really be clear as an acceptance of an invite.
@Askir what do you think?

Reply:
I am renaming it to InvitesView for now, but I am not sure if that conveys the right idea.

Reply:
InvitesView seems like a pretty general name that we'd use for all Restful resources.
However the plural view (InvitesView, UsersView, ArtifactsView) is always mounted under the colleciton path (/invites, /users, /artifacts). Since this handles the POST for a single Invite it should be `InviteView`.

Post on a single member of a resource is usually not defined for REST (https://www.restapitutorial.com/lessons/httpmethods.html). Some APIs use it as an alternative to PUT. But using it for a custom event is perfectly fine as long as it makes sense from an understanding point of view.